httpserver/external/binaries-list

```diff
@@ -14,4 +14,4 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-CD5B5996B46CB8D96C8F0F89A7A734B3C01F3DF7 tomcat-webserver-3.2.jar
+FCE6D146C0B03EF115B9AF05FF2976F15308998C tomcat:webserver:3.2.4
```

Comment:
Did you check in any way, that this still works? What is the difference between the jars?

Reply:
Version 3.2.4 is unique version in maven public https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22webserver%22

It's similiar to 3.2, adds a new package org.apache.tomcat.util.compat

Build code, run with ant tryme, update IDE, install Java EE plugins, add Liferay Tomcat server, config it, run it, stop it, without problems like a my Netbeans 8.2 installed version.